styles.js

```diff
@@ -9,6 +9,68 @@ export default createGlobalStyle`
 
   body {
     margin: 0;
-    font-family: system-ui;
+    font-family: fantasy;
+    background-color: #00a693;
+    margin-top: 100px;
+    padding: 15px;
+    color: #59311f;
+  }
+
+  h1 {
+    padding-top: 15px;
+    text-align: center;
+    border: dotted;
+    font-size: 40px;
+    margin: auto;
+    position: fixed;
+    top: 10px;
+    left: 50%;
+    translate: -50%;
+    background-color: #dbf3e1;
+    width: 95%;
+    color: #dbb290;
+    border-radius: 15px;
+    letter-spacing: 5px;
+  }
+
+  h2 {
+    text-align: center;
+    margin: auto;
+    text-decoration: underline;
+    width: 100%;
+    left: 0;
+    right: 0;
+    margin-bottom: -10px;
+    color: #dbf3e1;
+    font-weight: bold;
+    font-style: inherit;
+    letter-spacing: 1px;
+  }
```

Comment:
Normally, in the `styles.js` we only write very general styles.

These very specific styles work here for now because you only have one `h1` and one `h2`.  You will soon consider to move these styles to the relevant components so that you can choose the exact styling of the `h2` whereever you need it.